src/components/textbox/textbox.spec.tsx

```diff
@@ -68,37 +68,81 @@ describe("Textbox", () => {
   );
 
   describe(`when the characterLimit prop is passed`, () => {
-    it.each([2, 3, 4])("renders a character counter", (characterLimit) => {
+    let wrapper: ReactWrapper;
+    let characterLimit = 4;
+    let hintString =
+      "this is a test string where the limit will be nested here {}";
```

Comment:
**suggestion**: likewise I would define these locally in each test to help with readability, especially since not every test in this block uses all three variables.